### PR TITLE
fix(DB/Creature):improved Defender of Timbermaw

### LINKED ISSUE #3521: [Defender of Timbermaw] Doesn't heal
##### CURRENT BEHAVIOUR:
Use of the trinket [[Defender of Timbermaw]](https://classic.wowhead.com/item=21326) will summon a helper that will assist in melee combat. It does not heal, I've tried going to 1% hp in hopes it would trigger the healing. 

##### EXPECTED BLIZZLIKE BEHAVIOUR:
I never had this on retail. But the description says "Use: Calls forth a Timbermaw Ancestor to fight at your side and heal you. (10 Min Cooldown)"

##### STEPS TO REPRODUCE THE PROBLEM:
.additem 21326

##### EXTRA NOTES:
Same issue when i got the trinket on Warmane.

Commit: e1bf18374124ff05ab41f8014f001d27af1ab865
Server OS: Debian 10
Modules: None
Other customizations: None

<bountysource-plugin>

---
Want to back this issue? **[Post a bounty on it!](https://www.bountysource.com/issues/93265791-defender-of-timbermaw-doesn-t-heal?utm_campaign=plugin&utm_content=tracker%2F40032087&utm_medium=issues&utm_source=github)** We accept bounties via [Bountysource](https://www.bountysource.com/?utm_campaign=plugin&utm_content=tracker%2F40032087&utm_medium=issues&utm_source=github).
</bountysource-plugin>

### PR DESCRIPTION
**Pull request by Knindzagxg**
Fix for Defender of Timbermaw.

On spawn it will cast heal.
While active, it will help with few Lighting Bolts on random enemies.

To test:

.additem 21326
.mod rep 576 50000

Pull few mobs and damage then just a little to have agrro.
Use trinket and wait for heal / damage on them.

Should close #3521

### CODE REVIEW

**Comment by r0m1ntik94** [comment]
Work

**Review by FrancescoBorzi — APPROVED** [review]

**Comment by wizzymore** [comment]
It works but i don't think this is the correct behaviour:
https://classic.wowhead.com/item=21326/defender-of-the-timbermaw#comments:id=2711857
" - He will heal party members if they are below half health." Currently it heals everyone.
Also:
https://www.youtube.com/watch?v=lIYbQ6QA-DI&t=1m48s

**Comment by Knindzagxg** [comment]
I will check more into, but wasnt able to find about party.
Will do more tests today after work.

File: data/sql/updates/pending_db_world/rev_1601396210900255400.sql [hunk 1]
```diff
@@ -0,0 +1,15 @@
+INSERT INTO `version_db_world` (`sql_rev`) VALUES ('1601396210900255400');
+/*
+ * Zone: Defender of Timbermaw
+ * Update by Knindza | <www.azerothcore.org>
+ * Copyright (C) <www.shadowburn.net> & <www.lichbane.com>
+*/
+
+/* SMARTSCRIPT */
+UPDATE `creature_template` SET `AIName` = 'SmartAI' WHERE `entry` = 15720;
+UPDATE `creature_template` SET `unit_class` = '8' WHERE `entry` = 15720;
```
Comment (Helias) [inline data/sql/updates/pending_db_world/rev_1601396210900255400.sql:10]: don't use strings when integers are expected

**Review by Helias — CHANGES_REQUESTED** [review]

**Comment by Knindzagxg** [comment]
Updated.

On respawn will cast Heal on owner as we had before.
During fight, it will assist with 3-5 Lighting bolts and will cast owner or someone in party if bellow 50% hp.

**Review by FrancescoBorzi — APPROVED** [review]

**Review by Si1ker — APPROVED** [review]

**Review by Helias — APPROVED** [review]

**Pull request merged**